include/zephyr/dt-bindings/pinctrl/realtek-rts5912-pinctrl.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+ * Author: Lin Yu-Cheng <lin_yu_cheng@realtek.com>
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_REALTEK_RTS5912_PINCTRL_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_REALTEK_RTS5912_PINCTRL_H_
+
+#include <zephyr/dt-bindings/dt-util.h>
+
+#define REALTEK_RTS5912_GPIO_INOUT     BIT(0) /* IN/OUT : 0 input 1 output */
+#define REALTEK_RTS5912_GPIO_PINON     BIT(1) /* Input_detect : 1 enable 0 disable */
+#define REALTEK_RTS5912_GPIO_VOLT      BIT(2) /* Pin Volt : 1 1.8V 0 3.3V */
+#define REALTEK_RTS5912_FUNC0          0      /* GPIO mode */
+#define REALTEK_RTS5912_FUNC1          BIT(8) /* Function mode use BIT0~2 */
+#define REALTEK_RTS5912_FUNC2          BIT(9)
+#define REALTEK_RTS5912_FUNC3          ((BIT(8)) | (BIT(9)))
+#define REALTEK_RTS5912_FUNC4          BIT(10)
+#define REALTEK_RTS5912_GPIO_DRIVING   BIT(11) /* DRIVING : 0 low  1 high  */
+#define REALTEK_RTS5912_GPIO_SLEW      BIT(12) /* SLEW : 0 fast  1 slow  */
+#define REALTEK_RTS5912_GPIO_PULLDN    BIT(13) /* PULL DOWN : 0 disable 1 ebable */
+#define REALTEK_RTS5912_GPIO_PULLUP    BIT(14) /* PULL UP : 0 disable 1 ebable */
+#define REALTEK_RTS5912_GPIO_SCHMITTER BIT(15) /* Schmitter : 1 ebable 0 disable */
+#define REALTEK_RTS5912_GPIO_TYPE      BIT(16) /* TYPE : 0 PP 1 OD */
+#define REALTEK_RTS5912_GPIO_HILOW     BIT(17) /* OUTPUT : 0 low 1 high */
+
+#define REALTEK_RTS5912_INPUT_OUTPUT_POS    0
+#define REALTEK_RTS5912_INPUT_DETECTION_POS 1
```

Comment:
same here: these are props that need to be managed using standard props.

Reply:
In newest version, we support using dts to config these props by standards props, thanks.